v4/mobile/react-native/FastLink/ios/Podfile

```diff
@@ -1,49 +1,6 @@
 platform :ios, '11.0'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-def add_flipper_pods!(versions = {})
```

Comment:
Most of the Flipper integration code could be substituted by the usage of the `use_flipper` script, but this requires upgrading `react-native` (https://fbflipper.com/docs/getting-started/react-native-ios/#react-native-063).